setup.py

```diff
@@ -17,16 +18,39 @@ def version():
     return module.__version__
 
 
+def is_hiredis_bundled():
+    hiredis_submodule = 'vendor/hiredis'
```

Comment:
We also support windows. How about something like the following, just in case.

```
hiredis_submodule = os.path.join('vendor', 'hiredis')
if os.path.exists(hiredis_submodule)
```

Reply:
I developed with Python on Windows 8 years ago and I remember forward slashes as correctly understood by Python on Windows as well as backslashes; this link suggests that's indeed the case: https://stackoverflow.com/a/12288763/2896799